doc: matter: Removed deprecated Android Chip Tool sections (#10189)
nrfconnect/sdk-nrf: doc/nrf/ug_matter_overview_network_topologies.rst

* Regular IPv6 communication after the accessory device joins the Thread or Wi-Fi network - to interact with each other by exchanging application messages.
  For example, to report temperature measurements of a sensor.

> 

Reviewer: Please edit line 93 to:
```
The following figure shows the supported Matter controller types in the |NCS|.
```

> The CHIP Tool for Linux or macOS can be used as a Matter controller for testing Matter applications based on the |NCS|:
> That controller type is compatible with the |NCS| implementation of Matter.

Reviewer: ```suggestion
For testing Matter applications in the |NCS|, we recommend using the CHIP Tool for Linux or macOS as the Matter controller, which is compatible with the |NCS| implementation of Matter.
```

> For information about how to build and configure them, see the pages in the :ref:`ug_matter_gs_testing` section.

Reviewer: ```suggestion
For information about how to build and configure this controller, see the pages in the :ref:`ug_matter_gs_testing` section.
```
